modules/installation-osp-external-lb-config.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: REFERENCE
+[id="install-osp-external-lb-config_{context}"]
+= Installation configuration for a cluster on OpenStack with a user-managed load balancer
+
+:FeatureName: Deployment on OpenStack with User-Managed Load Balancers
+include::snippets/technology-preview.adoc[]
+
+The following example `install-config.yaml` file demonstrates how to configure a cluster that uses an external, user-managed load balancer rather than the default internal load balancer.
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: mydomain.test 
+compute:
+- name: worker
+  platform:
+    openstack:
+      type: m1.xlarge
+  replicas: 3
+controlPlane:
+  name: master
+  platform:
+    openstack:
+      type: m1.xlarge
+  replicas: 3
+metadata:
+  name: mycluster
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 192.168.10.0/24
+platform:
+  openstack:
+    cloud: mycloud
+    machinesSubnet: 8586bf1a-cc3c-4d40-bdf6-c243decc603a <1>
+    apiVIPs:
+    - 192.168.10.5
+    ingressVIPs:
+    - 192.168.10.7
+    loadBalancer:
+      type: UserManaged <2>
+featureSet: TechPreviewNoUpgrade <3>
+----
+<1> Regardless of which load balancer you use, the load balancer is deployed to this subnet.
+<2> The `UserManaged` value indicates that you are using an user-managed load balancer.
+<3> While user-managed load balancers are in Technology Preview, you must include the `TechPreviewNoUpgrade` value to deploy a cluster that uses a user-managed load balancer.
```

Comment:
s/While/Because

Reply:
We should add a warning around setting the `TechPreviewNoUpgrade` feature set.

```
[WARNING]
====
Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. You should not enable this feature set on production clusters.
====
```

Could also link to https://docs.openshift.com/container-platform/4.11/nodes/clusters/nodes-cluster-enabling-features.html

Reply:
```suggestion
<3> Because user-managed load balancers are in Technology Preview, you must include the `TechPreviewNoUpgrade` value to deploy a cluster that uses a user-managed load balancer.
```

Reply:
@pierreprinetti Just to confirm, is this a `TechPreviewNoUpgrade` feature, and should it/will it be included in the linked source code?

Reply:
yes it's TechPreview and no it's not included in this part of the code. I'm not sure exactly why, I can ask @mandre who probably knows more.

Reply:
@EmilienM Yeah, I think that's prudent. Thanks.

Reply:
@EmilienM Any update here?

Reply:
Martin is on PTO, let me ask around.

Reply:
Also TODO: evaluate inclusion in https://docs.openshift.com/container-platform/4.13/nodes/clusters/nodes-cluster-enabling-features.html

Reply:
@maxwelldb @mburke5678 the external LB feature is released as Tech Preview for now and this is expected that to enable this feature we need to enable `TechPreviewNoUpgrade` featureset.
Indeed it is not included in the linked source code but that should not block this documentation patch to move forward.

We are aware that activating the TechPreviewNoUpgrade features via the FeatureGate has serious ramifications for the users, but this is expected I believe for now. The feature should be GA in 4.14 for the OpenStack platform: https://issues.redhat.com/browse/OSASINFRA-3101

Reply:
@EmilienM Added https://github.com/openshift/openshift-docs/pull/57520/commits/4234501079471e05655c2e186c52f360bcdda78d for this. Is there a value that I can use in place of `REPLACE_ME`?

Reply:
Response: https://github.com/openshift/openshift-docs/commit/4234501079471e05655c2e186c52f360bcdda78d#r111047710

Reply:
@maxwelldb If this feature gets added to the TechPreviewNoUpdate feature set, please let me know. I have a PR to update the list of features in that set. I can add it there, if needed.

Reply:
@mburke5678 Have you seen features added to that list that are not part of https://github.com/openshift/api/blob/release-4.13/config/v1/types_feature.go#L111 and don't propose an ID?

Reply:
@maxwelldb I confusing myself (again!). It appears that user needs only to set the `featureSet: TechPreviewNoUpgrade` parameter here, and not [enable the `TechPreviewNuUpgrade` feature set, here](https://docs.openshift.com/container-platform/4.12/nodes/clusters/nodes-cluster-enabling-features.html#nodes-cluster-enabling-features-console_nodes-cluster-enabling). 
It struck me as odd that they use the same string. 
I _think_ we are all set here.

Reply:
@EmilienM Just to confirm, mburke's correct in his explanation here? ☺️

Reply:
yes, we are all set

Reply:
Groovy. I'll revert unneeded stuff on Monday.